c_src/opengl_video_compositor/shaders/vertex.glsl

```diff
@@ -0,0 +1,13 @@
+R"(
+#version 330 core
+
+layout (location = 0) in vec3 aPos;
+layout (location = 1) in vec2 aTexCoord;
+
+out vec2 texCoord;
+
+void main() {
+    gl_Position = vec4(aPos, 1.0);
+    texCoord = aTexCoord;
+}
+)"
```

Comment:
Configure your editor to make a two spaces indentation and add a newline at the end of a file.

Reply:
Shouldn't we specify this at the project level rather than in our editors? This doesn't apply here, since this is a glsl file, but I mean having a .clang-format file for C/C++

Reply:
Nevermind actually, both of these can be configured with a .editorconfig and we have one already